views/lobby.ejs

```diff
@@ -88,11 +88,11 @@
 					</div>
 				</div>
 
-				<span class='gun gunBefore'><img src='pictures/gun.png'></span>
-				<span class='gun gunBefore'><img src='pictures/gun.png'></span>
-				<span class='gun gunBefore'><img src='pictures/gun.png'></span>
-				<span class='gun gunBefore'><img src='pictures/gun.png'></span>
-				<span class='gun gunBefore'><img src='pictures/gun.png'></span>
+				<span class='gun gunBefore'><img class="gunImg" src='pictures/gun.png' style="max-height: 80%; max-width: 80%;"></span>
```

Comment:
Should all gun images have max-height and max-width set to 80% for both shields and gun displays?

Reply:
I got lazy to be honest. I scaled the image down via code instead of scaling the image file itself. Sorry and thanks!

Accepted. Will be resolved in the next commit

Reply:
All good! Thanks :D